react/ListPriceRange.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext } from 'react'
+import { defineMessages } from 'react-intl'
+import { ProductContext } from 'vtex.product-context'
+import { FormattedCurrency } from 'vtex.format-currency'
+import { useCssHandles } from 'vtex.css-handles'
+import { IOMessageWithMarkers } from 'vtex.native-types'
+
+import { StorefrontFC, PriceRangeProps } from './types'
+
+const CSS_HANDLES = [
+  'listPriceRange',
+  'listPriceRangeMinValue',
+  'listPriceRangeMaxValue',
+] as const
+
+const ListPriceRange: StorefrontFC<PriceRangeProps> = props => {
+  const { message, noRangeMessage, markers } = props
+  const handles = useCssHandles(CSS_HANDLES)
+  const { product } = useContext(ProductContext)
+
+  const priceRange = product?.priceRange
+  if (!priceRange) {
```

Comment:
⛏ [Nitpick] Why not add this conditional to the one below it? Would that `if` become too bloated?